### PR TITLE
Get and Set System Time

### PR DESCRIPTION
**Pull request by rahulmah**
Added test case to test following scenarios
1. Get system time using ipmi
2. Set valid system time
3. Set invalid system time
4. Set valid time on a system with epoch time.

### CODE REVIEW

File: lib/ipmi_client.robot [hunk 1]
```diff
@@ -20,6 +20,7 @@ Run IPMI Command
 Run IPMI Standard Command
     [arguments]    ${args}
     ${output}=  Execute Command    /tmp/ipmitool -I dbus ${args}
+    Return From Keyword If    '${output}' == ''    error
```
Comment (mkumatag) [inline lib/ipmi_client.robot]: output being empty does not mean that your command really has failed. Instead I would add a code to handle the error based on the exit status(return code) for the command. 

I would consider playing around these arguments to the execute commands -  return_stdout=True, return_stderr=True, return_rc=True and then check the rc value.

**Comment by rahulmah** [comment]
Made changes as per review comment from Manjunath.  Changes are there in above commit "Made changes to better handle IPMI standard command failure."

**Comment by rahulmah** [comment]
Made change to display error on failure of IPMI Standard command.

**Comment by mkumatag** [comment]
Looks good, merging the code changes.

**Pull request merged**